app/controllers/users_controller.rb

```diff
@@ -10,12 +10,10 @@ def create
       flash[:success] = "Welcome, #{@user.name}"
       redirect_to '/profile'
     elsif User.email_exists?(@user.email)
-      flash[:failure] = @user.errors.full_messages.to_sentence
       @user.email = nil
-      render :new
+      user_fails_create_validation
```

Comment:
Love this refactor! Why not always have our errors spit out!

Reply:
Yeah I think there's definitely some more areas where we can refactor and make helper/private methods within our controllers!